zk/tests/zkvm/methods/Cargo.toml

```diff
@@ -4,7 +4,7 @@ version = "0.1.0"
 edition = "2021"
 
 [build-dependencies]
-risc0-build = { workspace = true }
+risc0-build = { version = "1.1.2" }
```

Comment:
Is there a reason to not update the version at the workspace level and inherit it here?

Reply:
Simply adopted this from RiscZero